64 criar fucionalidade get pets usuário e update profile do usuário.

LeviUNIFOR opened the pull request:
close #64

LeviUNIFOR committed:
64 criar fucionalidade get pets usuário e update profile do usuário.

breno174 commented on petconnect/src/app/(drawer)/home/chatscreenconversation/[userId].tsx:
Temos um PR para alterar essa branch e voltar a exibir a tela de chat.

breno174 committed:
Update [userId].tsx (#67)

update file to catch enviroments variavebles and fix the error with dotenv lib.

breno174 reviewed (approved):
close #65
